docs/cases/api/cases-api/cases-api-create.asciidoc

```diff
@@ -27,7 +27,7 @@ configuration. |Yes
 settings. |Yes
 |`owner` |String |The application that owns the case. |Yes
 
-Must be a valid application owner that is registered within the Cases Role Based Access Control system (i.e. `securitySolution`).
+Must be a valid application owner that is registered within the Cases Role-based Access Control system (`securitySolution`).
```

Comment:
Looking around the Elastic docs, all-lowercase seems to be the prevalent usage for "role-based access control."
```suggestion
Must be a valid application owner that is registered within the Cases role-based access control system (`securitySolution`).
```